.github/workflows/rust.yml

```diff
@@ -202,8 +202,11 @@ jobs:
       - name: Checkout source code
         uses: actions/checkout@v2
 
-      - name: Remove default members
-        run: sed '/default-members/d' -i ${{ matrix.directory }}/Cargo.toml
+      # Store `Cargo.toml` before modifying it to not affect the diff.
+      - name: Remove and store default members
+        run: |
+          cp "${{ matrix.directory }}/Cargo.toml" "${{ matrix.directory }}/Cargo.toml.orig"
```

Comment:
```suggestion
      - name: Remove and store default members
        run: |
          # Backup `Config.toml` to restore it later so that `git diff` does not report any changes
          cp "${{ matrix.directory }}/Cargo.toml" "${{ matrix.directory }}/Cargo.toml.orig"
```